lib/matplotlib/tests/test_contour.py

```diff
@@ -177,7 +177,7 @@ def test_contour_manual_labels():
     x, y = np.meshgrid(np.arange(0, 10), np.arange(0, 10))
     z = np.max(np.dstack([abs(x), abs(y)]), 2)
 
-    plt.figure(figsize=(6, 2))
+    plt.figure(figsize=(6, 2), dpi=200)
```

Comment:
Why didn't this make the test images bigger?

Reply:
umm, I think because the printed version doesn't care about figure dpi?   There must be a way to change the test image dpi....  I'll check  Thanks

Reply:
Fixed

Reply:
How big size wise is the new image?  Would we get the same effect going down?